contrib/fuzz/seed_corpus_from_chain.py

```
@@ -0,0 +1,326 @@
+
+def extract_governance_objects(output_dir, datadir=None):
+    """Extract governance objects (proposals, triggers)."""
+    print("Extracting governance objects...")
+    result = dash_cli("gobject", "list", "all", datadir=datadir)
+    if not result:
+        return 0
+
+    saved = 0
+    try:
+        objects = json.loads(result)
+        for obj_hash, obj_data in objects.items():
```

Ruff (0.15.1)

[warning] Loop control variable `obj_hash` not used within loop body

Rename unused `obj_hash` to `_obj_hash`

(B007)